### PR TITLE
feat: Mock builder updates

### PR DESCRIPTION
**Pull request by marioevz**
## Changes included
- Modifies the static key included in https://github.com/marioevz/mock-builder/pull/7
- Adds log level
- Adds the cancun timestamp to the tx spammer, requires https://github.com/MariusVanDerWijden/tx-fuzz/pull/28

Opening PR as draft because the changes are not yet in the docker image referenced by the file

### CODE REVIEW

**Review by parithosh — APPROVED** [review]

File: src/mock_mev/mock_mev_launcher.star [hunk 1]
```diff
@@ -14,8 +14,9 @@ def launch_mock_mev(plan, el_uri, beacon_uri, jwt_secret):
 			cmd = [
 				"--jwt-secret={0}".format(jwt_secret),
 				"--el={0}".format(el_uri),
-				"--cl={0}".format(beacon_uri)
+				"--cl={0}".format(beacon_uri),
+				"--log-level=debug"
```
Comment (parithosh) [inline src/mock_mev/mock_mev_launcher.star:18]: Do you want to hardcode `debug` or use the value of `global_client_log_level`?

Comment (marioevz) [reply src/mock_mev/mock_mev_launcher.star:18]: Oh that would be much better! Let me change it :)

**Comment by parithosh** [comment]
The image with the tx-fuzz fix has been merged. New image has been released. https://github.com/MariusVanDerWijden/tx-fuzz/pull/28

**Comment by parithosh** [comment]
I guess this includes the tx-fuzz update from marius?

**Comment by marioevz** [comment]
> I guess this includes the tx-fuzz update from marius? I've created an issue (@barnabasbusa will take care of it as im on vacation this week) - so that'd be our long term solution for it. for reference: #157

It doesn't but don't worry about it, I'm using this one to file issues to the clients regarding the builder api

> I guess this includes the tx-fuzz update from marius? I've created an issue (@barnabasbusa will take care of it as im on vacation this week) - so that'd be our long term solution for it. for reference: #157

FYI It doesn't contain the update from Marius, I just rebased because Teku was failing on the branch I was basing off.

**Pull request closed without merging**